.travis.yml

```diff
@@ -17,10 +17,6 @@ stages:
 services:
   - docker
 
-cache:
```

Comment:
why not?

Reply:
This is mentioned in the commit message of https://github.com/tediousjs/tedious/pull/803/commits/e5e9387cf0c69975f68574107eeeda15c52a6b48: When a `package-lock.json` exists in the repo, Travis performs always a "clean installation" of dependencies that deletes the `node_modules` folder always, thus completely negating the caching of the folder.

Reply:
grrr, Travis being stupid. One more reason why CircleCI is so much better :D